### DISCUSSION TITLE
Security Onion and Proxmox Port Mirroring

### DISCUSSION THREAD

**Discussion by 5MI7th3MI6** [Security-Onion-Solutions/securityonion#8956] [Unsupported Versions]
Hello, 

I am setting up Security Onion on Proxmox and the switch I am using on my network supports port mirroring so do I need to configure port mirroring on the software side (Like mentioned in this discussion https://github.com/Security-Onion-Solutions/securityonion/discussions/8245 )of Proxmox or should the physical set be enough?

**Comment by weslambert** [comment]
It's been a while since I've looked at it, but you may need to perform some additional configuration to make it work properly, as it may not handle the traffic properly at the PM virtual interface.

**Comment by isaacgolding** [comment]
I just re-setup my system last week and still had to setup the port mirroring trick as referenced in [#8245](https://github.com/Security-Onion-Solutions/securityonion/discussions/8245) There are other methods to achieve the same goal with proxmox but this one seemed to be the easiest.
